Sources/StreamChat/StateLayer/Chat.swift

```diff
@@ -606,6 +624,57 @@ public final class Chat {
         try await messageUpdater.translate(messageId: message, to: language)
     }
     
+    // MARK: - Moderation
+    
+    /// Bans the specified user from the channel.
+    ///
+    /// When the user is banned, they will not be allowed to post messages until the ban is removed
+    /// or expired but will be able to connect to Chat and to channels as before.
+    ///
+    /// Learn more about [banning and moderation tools](https://getstream.io/chat/docs/ios-swift/moderation/?language=swift#ban).
+    ///
+    ///  - Note: Channel watchers cannot be banned.
+    ///  - Note: In most cases, only admins or moderators are allowed to ban other users from the channel.
+    ///
+    /// - Parameters:
+    ///   - userId: The user id of the channel member.
+    ///   - reason: The reason that the ban was created.
+    ///   - timeoutInMinutes: The number of minutes the user should be banned for. Nil means that the user is banned forever or until the user is unbanned explicitly.
+    ///
+    /// - Throws: An error while communicating with the Stream API or missing required capabilities.
+    public func banUser(_ userId: UserId, reason: String? = nil, timeout timeoutInMinutes: Int? = nil) async throws {
```

Comment:
All the other platforms use the term `User` here. Therefore, I added `banUser` and not `banMember`.

Reply:
It depends on what is the scope of the ban - if it's only the channel, then a member is banned. If the user is banned throughout the chats, then user should be used. I think in this case member is correct.

Reply:
Changed it to "Member"